firefox-ios/Client/Frontend/ContextualHint/ContextualHintEligibilityUtility.swift

```diff
@@ -75,9 +73,7 @@ struct ContextualHintEligibilityUtility: ContextualHintEligibilityUtilityProtoco
     /// presented if the feature is enabled. If the Toolbar CFR flag is disabled or the device
```

Comment:
Looks like the comment here should be adjusted too, or with next comment we might need to remove this entirely